viewer/Viewer.cpp

```diff
@@ -140,6 +140,31 @@ namespace Enki
 		}
 	}
 	
+	void ViewerWidget::setCamera (QPointF pos, double altitude, double yaw, double pitch)
+	{
+		this->pos = pos;
```

Comment:
Note that the internal pos member has inverted values, it actually encodes (-x,-y), see Viewer.cpp:690 for its use.